kong/plugins/jwt/handler.lua

```diff
@@ -146,38 +146,44 @@ local function set_consumer(consumer, credential, token)
   kong.ctx.shared.authenticated_jwt_token = token -- TODO: wrap in a PDK function?
 end
 
+local function unauthorized(message, www_auth_content, errors)
+  return { status = 401, message = message, headers = { ["WWW-Authenticate"] = www_auth_content }, errors = errors }
+end
+
 
 local function do_authentication(conf)
   local token, err = retrieve_tokens(conf)
   if err then
     return error(err)
   end
 
+  local www_authenticate_base = conf.realm and fmt('Bearer realm="%s"', conf.realm) or 'Bearer'
+  local www_authenticate_with_error = www_authenticate_base .. ' error="invalid_token"'
```

Comment:
The [RFC6750 - The OAuth 2.0 Authorization Framework: Bearer Token Usage](https://datatracker.ietf.org/doc/html/rfc6750) describes that:

> All challenges defined by this specification MUST use the auth-scheme
   value "Bearer".  This scheme MUST be followed by one or more
   auth-param values.  The auth-param attributes used or defined by this
   specification are as follows.  Other auth-param attributes MAY be
   used as well.
   
So we MUST pass at least one auth-param value. Therefore I added `error="invalid_token"`. However it also states that for requests without JWT token at all it should not add error code.
 
 > If the request lacks any authentication information (e.g., the client
   was unaware that authentication is necessary or attempted using an
   unsupported authentication method), the resource server SHOULD NOT
   include an error code or other error information.

So I need two variables: `www_authenticate_base` to return for requests without `jwt` token and `www_authenticate_with_error`.